src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  height: 60px;
+  line-height: 60px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
+  text-decoration: none;
+  display: inline-block;
+  transition:
+    color 0.5s,
+    border-color 0.5s;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+}
+
+a.is-active {
+  position: relative;
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--main-color);
```

Comment:
The `var(--main-color)` variable is used here, but it is not defined in this CSS file. Make sure to define it in a `:root` selector or ensure it's defined elsewhere in your styles.